src/components/accessories/patientDataForm/PatientDataForm.tsx

```diff
@@ -98,6 +99,50 @@ const PatientDataForm: FunctionComponent<TProps> = ({
     }))
   );
 
+  const allAgeTypes = useSelector(
+    (state: IState) => state.ageTypes.getAllAgeTypes.data
+  );
+
+  const getBirthDateAndAge = (
+    ageType: string,
+    formattedValues: Record<string, any>
+  ): { birthDate: string; age: number } => {
+    if (ageType == "agetype") {
```

Comment:
I think that a switch case would be more appropriate for the code reading in this case

Reply:
Okay